hubconf.py

```diff
@@ -120,6 +122,12 @@ def radio_model(
         adaptor.head_idx = tidx
         adaptors[adaptor_name] = adaptor
 
+    feat_norm_sd = get_prefix_state_dict(state_dict, '_feature_normalizer.')
+    feature_normalizer = None
+    if feat_norm_sd:
+        feature_normalizer = FeatureNormalizer(feat_norm_sd['mean'].shape[0], dtype=dtype)
+        feature_normalizer.load_state_dict(feat_norm_sd)
```

Comment:
Do we need to load the state dict here? Can this not be done at the model's  top level?

Reply:
The normalizer is somewhat in purgatory. It's not part of the `base_model` since it's something I add after training. So this `.load_state_dict` is similar to the conditioner, which is extra state that we need to load, but technically not part of the base model, but rather the `radio` model which wraps it.